components/brave_rewards/resources/donate/components/siteBanner.tsx

```diff
@@ -126,6 +126,10 @@ class Banner extends React.Component<Props, State> {
       if (internalFavicon.test(publisher.logo)) {
         logo = `chrome://favicon/size/160@2x/${publisher.logo}`
       }
+
+      if (!publisher.verified) {
+        logo = ''
```

Comment:
This seems like something we should handle in the ledger when we fetch banner info instead of blank on the client side, wdyt?

Reply:
@ryanml for now I would prefer to keep the logic and just not display it as we don't know where this will take us

Reply:
Fair enough!